tests/manage/cluster/test_add_mds_to_cluster.py

```diff
@@ -7,88 +7,76 @@
 
 from ocs_ci.ocs import constants, defaults, ocp
 from ocs_ci.framework import config
-from ocs_ci.framework.testlib import tier1, ManageTest, bugzilla
+from ocs_ci.framework.testlib import (
+    tier1, ManageTest, bugzilla, ignore_leftovers
+)
 from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility.utils import TimeoutSampler
 
 log = logging.getLogger(__name__)
 
-CEPHFS_DELETED = '"{cephfs_name}" deleted'
-
-POD = ocp.OCP(kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace'])
-CEPHFS = ocp.OCP(kind=constants.CEPHFILESYSTEM)
-CEPH_OBJ = None
-
-
-@pytest.fixture(scope='class')
-def test_fixture(request):
-    """
-    Create disks
-    """
-    self = request.node.cls
-
-    def finalizer():
-        teardown(self)
-    request.addfinalizer(finalizer)
-    setup(self)
-
 
-def setup(self):
+@pytest.fixture()
+def fs_setup(request):
     """
     Setting up the environment for the test
     """
-    global CEPHFS, CEPH_OBJ
-    CEPHFS = ocp.OCP(kind=constants.CEPHFILESYSTEM)
-    self.fs_data = CEPHFS.get(defaults.CEPHFILESYSTEM_NAME)
-    self.fs_name = self.fs_data['metadata']['name']
-    CEPH_OBJ = OCS(**self.fs_data)
-
+    def finalizer():
+        fs_data = cephfs.get(defaults.CEPHFILESYSTEM_NAME)
+        fs_data['spec']['metadataServer']['activeCount'] = (
+            original_active_count
+        )
+        ceph_obj.apply(**fs_data)
 
-def teardown(self):
-    """
-    Tearing down the environment
-    """
-    self.fs_data = CEPHFS.get(defaults.CEPHFILESYSTEM_NAME)
-    self.fs_data['spec']['metadataServer']['activeCount'] = (
-        self.original_active_count
-    )
-    CEPH_OBJ.apply(**self.fs_data)
-    assert verify_fs_exist(self.original_active_count * 2)
+    request.addfinalizer(finalizer)
+    cephfs = ocp.OCP(kind=constants.CEPHFILESYSTEM)
+    fs_data = cephfs.get(defaults.CEPHFILESYSTEM_NAME)
+    original_active_count = fs_data.get('spec').get('metadataServer').get('activeCount')
+    ceph_obj = OCS(**fs_data)
+    return original_active_count, fs_data, ceph_obj
 
 
-def verify_fs_exist(pod_count):
+def mds_active_count():
     """
-    Verifying if a ceph FS exist
+    Return Active MDS Count
```

Comment:
Specify `Returns:` section in the docstring and I see it returns tuple with new_active_count, pods so document this please.